protocol/javatests/org/openyolo/protocol/CredentialTest.java

```diff
@@ -69,6 +62,11 @@ public void testBuilder_emptyIdentifier() {
     }
 
     @Test(expected = RequireViolation.class)
+    public void testBuilder_emptyAuthMethod() {
+        new Credential.Builder(EMAIL_ID, "", "https://www.example.com");
```

Comment:
It's an incompatible type for this method call, but I've extracted AUTH_DOMAIN_STR for this purpose.